app/views/layouts/_flash.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="grid-row">
```

Comment:
Are we using flash messages yet? If not we do a lot less with them as we're using the form builder errors. I'd rather get rid of this and add a more up to date implementation when we need it.

Reply:
Removed.